### ISSUE TITLE
feature: Have a simple option to disable enable SSH for an account owner

### ISSUE THREAD

**Issue by shoulders** [virtualmin/virtualmin-gpl#703]
|  SYSTEM INFORMATION||
|----------------------|---------------------------|
|  OS type and version |     Ubuntu Linux 22.04.3     |
|  Webmin version      |     2.105      |
|  Usermin version     |     2.005     |
|  Virtualmin version  |     7.8.2       |
|  Theme version       |     21.09.5      |
|  Package updates     |      12  package updates are available   |

### The issue
There is no simple way to enable/disable SSH access for account owners.

you can use `Virtualmin --> System Customization -->custom shells` to set things globally

### Why
most amatuer hosters will just want SSH off for everyone and they need an easy way to set this. I am from the windows world so the `custom shells` is far beyond me but I know I should turn SSH of for most of my accounts, they dont need it.

### The proposed solution
- have an option in the sever template and corresponding option in the Virtual server somewhere so you can  toggle (enable/disable) SSH for the account owner. 
- The account owner should not be able to enable/disble this themselves.

**Comment by iliajie** [comment]
Hello, 

This is a good suggestion. As I'm working now on improving **Edit Users** page, so I don't mind adding a drop-down for having control over virtual server administrator user shell.

**Comment by shoulders** [comment]
I cant, the permissions are hardcoded. Also this initial setting should be settable in the server template.

![image](https://github.com/virtualmin/virtualmin-gpl/assets/319997/a1eb3fc7-4d63-4f59-8ca0-a348cc57f177)

`Virtualmin --> Administration Options --> Edit Owner Limits --> Other restrictions`

**Comment by jcameron** [comment]
You may need to designate another shell as usable by domain admins at System Customization -> Custom Shells.

Also, you can designate a default shell in the server templates, on the Administration user page.

**Comment by shoulders** [comment]
> You may need to designate another shell as usable by domain admins at System Customization -> Custom Shells.
> Also, you can designate a default shell in the server templates, on the Administration user page.

As an end user, I have no idea what these settings mean, hence the on/off button :smile:.

**Comment by iliajie** [comment]
> As an end user, I have no idea what these settings mean, hence the on/off button 😄.

Yes, I agree, it isn't simple to grasp for the normal users. We may change this though for master administrator to be able to create SSH users. I'm working on improving **Edit Users** page, which should address this issue as well.

**Comment by jcameron** [comment]
One easy improvement we could make would be to add a no-SSH shell as one of the default options for the domain admin user.

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run. @shoulders, sorry for the noise, and thanks for the contribution. We definitely didn't mean any disrespect.

If it turns out the issue is already resolved for you, you're welcome to close it.
